bottlecap/src/bin/bottlecap/main.rs

```diff
@@ -46,6 +46,9 @@ use bottlecap::{
 };
 
 use bottlecap::config::{AwsConfig, Config};
+use bottlecap::logs::flusher::Flusher;
```

Comment:
can we move these up into the main bottlecap import on line 18?

Reply:
Oh good catch! There was actually a double import too! I will see if clippy has a rule for that